site/specs/numbers.json

```diff
@@ -5838,404 +5910,392 @@
+            "description": "The 200 OK  indicates a successful download request.<br>\nThe response to the get on a filename will result in the file being prepared and delivered in a manner consistent with browser download.  The Response headers include the content-disposition header describing the file download."
           },
           "404": {
-            "description": "<p>a 404 indicates that the indicated file was not found in conjunction with the order id.</p>\n"
+            "description": "a 404 indicates that the indicated file was not found in conjunction with the order id."
           }
         }
       },
       "put": {
         "tags": [
           "/accounts"
         ],
-        "description": "<p>A PUT on the filename will update / replace the identified file id.  The format of the PUT is identical to that of the POST.<br>\nHeader settings typical of a valid upload are...<br>\n<code>\nHost: dashboard.bandwidth.com <br>\nAuthorization: Basic xxxxxxxxxxxxxxxxxxxx== <br>\nContent-Type: application/pdf <br>\nAccept: <em>/</em> <br>\nAccept-Encoding: gzip, deflate <br>\nAccept-Language: en-US,en;q=0.8 <br>\nCache-Control: no-cache <br>\n    <br>\n----WebKitFormBoundaryE19zNvXGzXaLvS5C <br>\nContent-Disposition: form-data; name=\"george\"; filename=\"Bandwidth Dashboard.pdf\" <br>\nContent-Type: application/pdf <br>\n    <br>\n    <br>\n----WebKitFormBoundaryE19zNvXGzXaLvS5C <br>\n</code></p>\n",
-        "operationId": "PUT /accounts/{accountId}/importTnOrders/{orderid}/loas/{fileid}",
+        "description": "A PUT on the filename will update / replace the identified file id.  The format of the PUT is identical to that of the POST.",
+        "summary": "Updating Import Tn order loas file",
+        "operationId": "UpdateImportTnOrderLoasFile",
         "parameters": [
           {
             "name": "accountId",
-            "description": "",
+            "description": "User's account ID",
             "in": "path",
             "required": true,
             "schema": {
-              "type": "string",
-              "default": null
+              "type": "string"
             },
-            "example": null
+            "example": "1234567"
           },
           {
-            "name": "orderid",
-            "description": "",
+            "name": "orderId",
+            "description": "Import Tn order's ID",
             "in": "path",
             "required": true,
             "schema": {
-              "type": "string",
-              "default": null
+              "type": "string"
             },
-            "example": null
+            "example": "093a9f9b-1a78-4e47-b6e2-776a484596f4"
           },
           {
-            "name": "fileid",
-            "description": "",
+            "name": "fileId",
+            "description": "File name id.",
```

Comment:
These really short descriptions probably shouldn't have a `.` in them

Reply:
And is it a file name id or file id?

Reply:
file id with file format - don't know why)

Reply:
ahh, I see.  If those are just path variables, maybe we should rename them to `fileName`?  That might be more clear from the user's perspective

Reply:
it's not really file name - when user upload file with some readable in response our api will return renamed file and this name is id

Reply:
I see.  Ok, this should be fine then.